Undefined name: 'data' in ./src/itoolkit/transport/direct.py

[flake8](http://flake8.pycqa.org) testing of https://github.com/IBM/python-itoolkit on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./src/itoolkit/transport/direct.py:83:20: F821 undefined name 'data'
            return data.decode('utf-8')
                   ^
./src/itoolkit/transport/direct.py:85:20: F821 undefined name 'data'
            return data
                   ^
2     F821 undefined name 'data'
2
```

__E901,E999,F821,F822,F823__ are the "_showstopper_" [flake8](http://flake8.pycqa.org) issues that can halt the runtime with a SyntaxError, NameError, etc. Most other flake8 issues are merely "style violations" -- useful for readability but they do not effect runtime safety.
* F821: undefined name `name`
* F822: undefined name `name` in `__all__`
* F823: local variable name referenced before assignment
* E901: SyntaxError or IndentationError
* E999: SyntaxError -- failed to compile a file into an Abstract Syntax Tree

--- a/src/itoolkit/transport/direct.py
+++ b/src/itoolkit/transport/direct.py
@@ -77,7 +77,7 @@ class DirectTransport(object):
         if not _available:
             raise RuntimeError("Not supported on this platform")
             
-        return _direct._xmlservice(itool.xml_in(), self.ctl, self.ipc)
+        data = _direct._xmlservice(itool.xml_in(), self.ctl, self.ipc)
         
         if sys.version_info >= (3,0):
             return data.decode('utf-8')

